apps/payments/next/app/[locale]/subscriptions/manage/en.ftl

```diff
@@ -25,3 +25,9 @@ subscription-management-iap-sub-expires-on = Expires on { $date }
 # $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
 subscription-management-button-manage-subscription-aria = Manage subscription for { $productName }
 subscription-management-button-manage-subscription = Manage
+## Interval of subscription's billing cycle, it will appear next to the product name (e.g. Mozilla VPN (daily), Monitor Plus (monthly))
+subscription-management-page-subscription-interval-daily = (daily)
```

Comment:
Are these pulled by another string or are they being concatenated in the code? If so, we should [avoid concatenation ](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#avoid-concatenations-use-placeholders-instead)and use a placeholder instead.